security.rst

```diff
@@ -449,6 +465,67 @@ you who you are and what roles you have:
 Because this URL requires ``ROLE_ADMIN``, if you had logged in as ``ryan``,
 this would deny you access. More on that later (:ref:`security-authorization-access-control`).
 
+.. tip::
+
+    If you have many providers and want to define the same encoder for all of
+    them, you can configure as follow:
+
+    .. configuration-block::
+
+        .. code-block:: yaml
+
+            # app/config/security.yml
+            security:
+                encoders:
+                    Symfony\Component\Security\Core\User\UserInterface: bcrypt
+
+                    # would be equivalent to:
```

Comment:
Let's use present tense: "is equivalent to:"

Reply:
Fixed.